pos_event_calendar/models/event_event.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2025 Moka
+# @author Damien Horvat <damien@moka.cloud>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+class EventEvent(models.Model):
+    """Custom model to load inside the POS"""
+
+    _name = "event.event"
+    _inherit = ["event.event", "pos.load.mixin"]
```

Comment:
```suggestion
    _inherit = "event.event"
```

Do not add the mixing again. It's already done by `pos_event` : https://github.com/odoo/odoo/blob/9463bfeb58fb40176ecf1131bac6627a1627d02c/addons/pos_event/models/event_event.py#L6